src/jdk.jdwp.agent/share/native/libjdwp/eventHandler.c

```diff
@@ -1252,6 +1258,46 @@ cbVMInit(jvmtiEnv *jvmti_env, JNIEnv *env, jthread thread)
     LOG_MISC(("END cbVMInit"));
 }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+/* Event callback for JVMTI_EVENT_VM_RESTORE */
+static void JNICALL
+cbVMRestore(jvmtiEnv *jvmti_env, ...)
+{
+    EventInfo info;
```

Comment:
```suggestion
    EventInfo info = {0};
```

Reply:
Please *don't* do this; there's already a use of `memset()` below.

Reply:
Reverted this change in https://github.com/ibmruntimes/openj9-openjdk-jdk/commit/1396459ab0d1f728e71ed55dd10af33c53c2c8b7 so that the object is just declared, since it will be memset to 0 anyways.